### ISSUE TITLE
Tabs component fails to call event.preventDefault() and event.stopPropagation() when handling keyboard interactions

### ISSUE THREAD

**Issue by brichwin** [indiana-university/rivet-source#602]
## Describe the bug
Keyboard events that are part of the IU Rivet dropdown component's interactivity are continuing to propagate and be handled by the browser as usual despite being handled by Rivet. This is causing undesirable effects. For example, using the home and end keys while focused on a tabs component's tab not only moves to the first or last tab as expected, but also errantly scrolls the page.

The undesirable effects (such as page scrolling) reduce IU Rivet's credibility and also could cause issues for users of screen-magnification as page scrolling could take the tabs component completely out of their zoomed in view.

## Expected behavior
Keystroke events handled by the tabs component should not cause the default browser reaction and probably should not continue bubbling up the event chain. The IU Rivet js code should at a minimum call `event.preventDefault()` for each keystroke event handled by a component. It's a good idea to also call `event.stopPropagation()`.

## Relevant context (required)
Here is [tabs.js code from the examples in the ARIA practices](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20161214/examples/tabs/js/tabs.js) document that correctly handles the home and end keys when focus is on a tab - note the calls to `event.preventDefault()`:
```js
 // Handle keydown on tabs
  function keydownEventListener (event) {
    var key = event.keyCode;

    switch (key) {
      case keys.end:
        event.preventDefault();
        // Activate last tab
        activateTab(tabs[tabs.length - 1]);
        break;
      case keys.home:
        event.preventDefault();
        // Activate first tab
        activateTab(tabs[0]);
        break;
...
```

## Steps to reproduce

1. Navigate to [Rivet 1.8.2 Default tabs example](https://rivet.iu.edu/components/page-content/tabs/#default-tabs-example) (I used Chrome on Windows 11 - Note, Safari on MacOS requires configuration before keyboard nav works)
2. Set the browser window to a height of around 2 inches (so page scrolling becomes obvious)
3. Move keyboard focus to the first tab ("Tab one")
4. Press the End key. Note that the page scrolls to the bottom.
5. Scroll back up and note that the last tab ("Tab three") did get focus.
6. Scroll the page such that the tabs in the default tabs example are at the very top.
7. Move keyboard focus to the last tab ("Tab three")
8. Press the Home key. Note that the page scrolls to the top.
9. Scroll if needed to note that the first tab ("Tab one") did get focus.

**Comment by brichwin** [comment]
This issue is present in Rivet 1 and Rivet 2.

**Comment by brichwin** [comment]
This issue also manifests in the dropdown component: https://github.com/indiana-university/rivet-source/issues/598

**Comment by levimcg** [comment]
This issue was addressed in #627 and is available in the [rivet-uits@1.8.3 release](https://github.com/indiana-university/rivet-source/releases/tag/v1.8.3)
